pkg/controller/nodemaintenance/nodemaintenance_controller.go

```diff
@@ -341,7 +345,7 @@ func (r *ReconcileNodeMaintenance) initMaintenanceStatus(nm *nodemaintenanceapi.
 	return nil
 }
 
-func (r *ReconcileNodeMaintenance) reconcileAndError(nm *nodemaintenanceapi.NodeMaintenance, err error) (reconcile.Result, error) {
+func (r *ReconcileNodeMaintenance) reconcileAndErrorExt(nm *nodemaintenanceapi.NodeMaintenance, err error, duration *time.Duration) (reconcile.Result, error) {
```

Comment:
- I'm not a big friend of these `ext`, `imp` etc postfixes in func names etc. Can you use something more descriptive please?
- I think even `reconcileAndError` is misleading, because it isn't reconciling anything. What about `handleError` and `handleErrorWithRequeueDuration`? A bit long but easier to understand on 1st look.

Reply:
reconcileAndError -> onReconcileError

Reply:
done